Tests/Microsoft.AppCenter.Test.Windows/AppCenterTest.cs

```diff
@@ -515,10 +515,68 @@ public void ParseAppSecretOnePlatformTerminatingSemicolon()
             var appSecret = Guid.NewGuid().ToString();
             var platformId = "uwp";
             var secrets = $"{platformId}={appSecret};";
-            var parsedSecret = AppCenter.GetSecretForPlatform(secrets, platformId);
+            var parsedSecret = AppCenter.GetSecretAndTargetForPlatform(secrets, platformId);
             Assert.AreEqual(appSecret, parsedSecret);
         }
 
+        /// <summary>
+        /// Verify parse when there is only one platform and both app secret and token
+        /// </summary>
+        [TestMethod]
+        public void ParseAppSecretAndTargetOnePlatformTerminatingSemicolon()
```

Comment:
The secret string doesn't contain a terminating semicolon.
```suggestion
        public void ParseAppSecretAndTargetOnePlatform()
```